src/scripts/components/DropdownMultiSelectAdvanced.jsx

```diff
@@ -49,7 +49,7 @@ class DropdownMultiSelectAdvanced extends React.Component {
   };
 
   handleToggle = () => {
-    if (this.state.isViewAllItems) {
+    if (this.state.isViewAllItems && this.state.searchText.length > 0) {
```

Comment:
remove this this.state.searchText.length > 0 condition from here and add below

Reply:
@yogitattn We need to call fetchAllItems function when searchText is available and it is working fine with these both conditions

Reply:
Yes, I know, I just want to move condition to below line, so that, whenever we move from one tab to another, we need to clear the text only, we are doing that already, but this will generate this issue again